### PR TITLE
disable xdebug, add PHP 5.6 and 7.0

### PR DESCRIPTION
**Pull request by SkySymbol**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,16 @@ language: php
...
+  - 7.0
 before_script:
   ## Composer
   - composer self-update
   - composer install --no-interaction --prefer-source
-  ## travis configuration
+  ## Travis configuration
   - cp app/config/config.ini-travis app/config/config.ini
+  ## Disabling xdebug
+  - phpenv config-rm xdebug.ini
```
Comment (flodolo) [inline .travis.yml]: I think you need to put it at the beginning of the `before_script` block

Comment (SkySymbol) [inline .travis.yml]: Done, thank you for the review

**Comment by flodolo** [comment]
Weird, the warning went away for ```php-cs``` but not for ```composer --version```, I assumed it was a order issue but it's not.

**Comment by flodolo** [comment]
Let's see if @pascalchevrel has ideas. Also not sure if he wants to keep 5.5 based on #553

**Comment by pascalchevrel** [comment]
yes, 5.5 should be removed.

**Pull request merged**